apps/omg/lib/omg/fees.ex

```diff
@@ -64,15 +64,15 @@ defmodule OMG.Fees do
   """
   @spec parse_file_content(binary()) :: {:ok, fee_t()} | {:error, list({:error, atom()})}
   def parse_file_content(file_content) do
-    {:ok, json} = Jason.decode(file_content)
-
-    {errors, token_fee_map, _} =
-      json
-      |> Enum.map(&parse_fee_spec/1)
-      |> Enum.reduce({[], %{}, 1}, &spec_reducer/2)
-
-    {Enum.reverse(errors), token_fee_map}
-    |> handle_parser_output()
+    with {:ok, json} <- Jason.decode(file_content) do
+      {errors, token_fee_map, _} =
+        json
+        |> Enum.map(&parse_fee_spec/1)
+        |> Enum.reduce({[], %{}, 1}, &spec_reducer/2)
+
+      {Enum.reverse(errors), token_fee_map}
+      |> handle_parser_output()
```

Comment:
is there a reason for discrepancy in 
`Jason.decode(file_content)` and 
```
{Enum.reverse(errors), token_fee_map}
|> handle_parser_output()
```
pipe or no pipe?